apps/src/templates/census2017/NameInput.jsx

```diff
@@ -0,0 +1,37 @@
+import React, {Component, PropTypes} from 'react';
+import i18n from "@cdo/locale";
+import {styles} from './CensusForm';
+
+class NameInput extends Component {
+
+  static propTypes = {
+    setName: PropTypes.func,
+    name: PropTypes.string
+  }
+
+  sendToForm(event) {
+    this.props.setName(event.target.value);
+  }
```

Comment:
nit: it's best to define callbacks like this:

sendToForm = (event) => {
  this.props.setName(event.target.value);
} 

which takes care of binding `this`.

Reply:
...you should do it this way for any function that you pass along as either a prop to another react component or as an event handler to a dom element.